GliaWidgets/Theme/Survey/Theme+Survey.swift

```diff
@@ -34,19 +34,24 @@ extension Theme {
         public var borderWidth: CGFloat = 0
         /// Button border hex color.
         public var borderColor: String?
+        /// Accessibility related properties.
+        public var accessibility: Accessibility
+
         /// Initializes `Button` style instance.
         public init(
             background: String,
             title: Theme.Text,
             cornerRadius: CGFloat,
             borderWidth: CGFloat = 0,
-            borderColor: String? = nil
+            borderColor: String? = nil,
+            accessibility: Accessibility = .unsupported
```

Comment:
Accessibility has to be required parameter and it is up to integrator to decide what to do with it.